etor/src/main/resources/openapi_etor.yaml

```diff
@@ -105,6 +105,38 @@ paths:
             application/json:
             schema:
               $ref: '#/components/schemas/NotFoundError'
+  /v1/etor/metadata/summary/{sender}:
+    get:
+      summary: Returns all the order and their status for particular hospital
```

Comment:
Is this metadata summary endpoint just for orders or will it work for results as well?

Reply:
results as in?

Reply:
Lab results. We have lab orders as one part of the flow from the hospital to the lab, and then we're adding lab results that will go from the lab to the hospital

Reply:
We can update this to include orders when or if we get to this point